source/_assets/sass/_base.scss

```diff
@@ -128,3 +132,8 @@ pre {
         @apply .p-0;
     }
 }
+
+::selection {
+    @apply .bg-blue-500;
+    @apply .text-white;
+}
```

Comment:
Could you add a line ending here?

And I think we intentionally didn't style the selection, so let's drop this unless this was a part of the upgrade process?

Reply:
I will fix the line ending, thanks.

I am pretty green at some of this. I had to look up ::selection.

Anyway, [those lines came from the 'blog' repo, here](https://github.com/tightenco/jigsaw-blog-template/blob/master/source/_assets/sass/_base.scss#L136)

Diomed originally put that in, and Damiani had touched/updated it also.

Should I take it out, or leave it?

Reply:
Ah! keep it! :) Thanks!!